lib/functions/compilation/kernel-debs.sh

```diff
@@ -473,6 +473,10 @@ function kernel_package_callback_linux_headers() {
 	[[ -f "${kernel_work_dir}/scripts/module.lds" ]] &&
 		run_host_command_logged cp -v "${kernel_work_dir}/scripts/module.lds" "${headers_target_dir}/scripts/module.lds"
 
+	# Ubuntu Resolute requires that there be a "scripts/mod/modpost" binary during header compilation
+	[[ -f "${kernel_work_dir}/scripts/mod/modpost" ]] &&
+		run_host_command_logged cp -v "${kernel_work_dir}/scripts/mod/modpost" "${headers_target_dir}/scripts/mod/modpost"
```

Comment:
The problem here is under cross-compilation: if kernel is for arm64 but is built say on amd64, the headers package will end up with a wrong architecture binary. 

Instead, we would need to ship enough sources so that the modpost binary can be built board-side during linux-headers install, like the others.

I've _very similar_ issue with `resolve_btfids`, but that one is easy since we _already_ ship enough sources to build it there.

Or: just wait until all Armbian kernels are 6.12+, and get rid of this whole packaging code (instead just do `make bindeb`).